.golangci.yaml

```diff
@@ -6,9 +6,9 @@ run:
 linters:
   enable:
     - errcheck
-    - govet
     - gosimple
+    - govet
+    - ineffassign
```

Comment:
Probably a good idea for us to keep these lists alphabetical as they grow larger.